services/search/main.py

```diff
@@ -0,0 +1,23 @@
+import testing.test_constants as constants
+from search_engine import SearchEngine
+
+def search_service(request):
+    """
+    Parses http json request and returns list of articles and their
+    scores depending on the input query
+    Args:
+        request: http POST body request as json
+    Returns:
+        list of articles with their respective scores
+    """
+    json = request.get_json()
+    query = json["query"]
+    search = SearchEngine(keywords_weight=2)
+    score_per_article = search.query(query)
+    
```

Comment:
Ideally, you can try/catch this call, and then return the formatted error here (You'll see that comment in other places too)